content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -17,6 +17,15 @@ You need [`read and write variables` permissions](/docs/cloud/users-teams-organi
 - The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
 - The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
 
+## Run-Specific Variables
+
+When using Terraform 1.1 or later, you can set input variable values for a particular plan/apply run on the command line using either of the following mechanisms:
+
+* Specifying `-var` and `-var-file` arguments
+* Local environment variables named `TF_VAR_` followed by the name of a declared variable
+
+Variable values set in either of these ways will take precedence over Workspace-Specific and Variable Set values. See the [Precedence](/docs/cloud/workspaces/variables.html#precedence) section for more details. For more information about specifying CLI input variable values, see the Terraform documentation for [Variables on the Command Line](https://www.terraform.io/docs/language/values/variables.html#variables-on-the-command-line) or the plan command [Input Variables on the Command Line](https://www.terraform.io/docs/cli/commands/plan.html#input-variables-on-the-command-line).
```

Comment:
I find it confusing that we have this documented in two places, and I bet users will, too :-) I find the language docs easier to read so I think we should just point folks there - it has all of the ways we identified above with clear headings and examples. LMK if you disagree!

```suggestion
Refer to the [variables on the command line](https://www.terraform.io/docs/language/values/variables.html#variables-on-the-command-line) documentation for more details and examples.
```